proposals/distribution.md

```diff
@@ -0,0 +1,156 @@
+
+    * What: Specifying authentication and authorization schemes
+    * In/Out/Future: Out of scope
+    * Status: N/A
+    * Description: Defining protocols for authenticating and authorizing distribution access.
+    * Why: As an HTTP-based protocol, clients and servers can negotiate authentication via HTTP's [challenge-response authentication framework][rfc7235-s2.1].
+        There is no need for the distribution specification to choose a particular authentication scheme, because clients receiving 401 and 407 responses can use IANA's [HTTP Authentication Scheme Registry][iana-auth] to look up referenced schemes and take appropriate action.
+
+* “Creating a reference spec for optional DNS based naming and discovery”.
+    Discovery and registry protocols are completely separate and do not need to be added together.
+    This entry replaces part of the previous “Creating Reference spec for optional DNS based naming & distribution” entry.
+
+    * What: Creating a reference spec for optional DNS based naming and discovery
+    * In/Out/Future: In scope for future specification
+    * Status: Work not yet started
+    * Description: Define a protocol for resolving an image name to retrieval information.
+        When we address this, we will also allow for alternative name-to-image discovery protocols in parallel with the OCI-specified protocol.
+    * Why: It is reasonable to provide a standardized way to use DNS based distribution in conjunction with OCI without requiring its use.
+        There are many good use cases for DNS based distribution, but not all use cases support this.
+        Furthermore, encoding the location of a bundle into the bundle can cause issues with downloads from alternate locations other than the origin specified in the name.
+
+* “Specifying a distribution method”.
+    This entry replaces part of the previous “Creating Reference spec for optional DNS based naming & distribution” and “Standardizing on a particular Distribution method” entries.
+
+    Retrieving image indexes covers the current “tag listing” (e.g. “what named manifests are in `library/busybox`?”), because tags are entries in the image format's [`manifests` array][manifests].
+    Other tag-listing endpoints needed for backwards-compatibility are therefore in scope as well.
+
+    Grouping image indexes in repositories is considered part of distribution policy or content management, which are out of scope for this entry's per-image action.
+    For example, “what images are under `library/`?” is out of scope for this project.
+
+    * What: Specifying a distribution method
+    * In/Out/Future: In scope
+    * Status: In progress (see opencontainers/distribution-spec)
+    * Description: Define a protocol for creating, retrieving, updating, and deleting objects defined in the [image specification][image-spec].
+        Listing repositories (like [`/v2/_catalog`][catalog]) is a multi-[image-index][] action, which is out of scope for this entry.
```

Comment:
Again, there is not relation to "multi-image-index" and the catalog endpoint. Please avoid making incorrect statements.

Reply:
> Again, there is not relation to "multi-image-index" and the catalog endpoint. Please avoid making incorrect statements.

How would you prefer to phrase it?  Do you agree that `/v2/_catalog` is a registry-level operation?  Do you agree that it may return multiple repositories?  Do you agree that repositories may contain multiple images?

Reply:
I have no clue what is being said here. It literally makes no sense. The specification makes no reference to a "multi-image-index" and we don't have anything called that in the image-spec. It is just made up.

Reply:
Is this possibly confusion caused by a misunderstanding of ManifestLists?

Reply:
These are unrelated concepts.

Reply:
Rereading the sentence, it sounds there's an assumption that the registry implements a 'multi-image index' (e.g. an indexed repository table in a relational database) used to serve the catalog endpoint. This document should avoid making any assumptions about the registry implementation.